packages/js/src/ui/components/elements/InboxStatus/InboxStatusOptions.tsx

```diff
@@ -53,10 +53,7 @@ export const StatusItem = (props: {
   const style = useStyle();
 
   return (
-    <Dropdown.Item
-      class={style('inboxStatus__dropdownItem', cn(dropdownItemVariants(), 'nt-flex nt-gap-8'))}
-      onClick={props.onClick}
-    >
+    <Dropdown.Item appearanceKey="inboxStatus__dropdownItem" class="nt-flex nt-gap-8" onClick={props.onClick}>
```

Comment:
How come we don't need the `dropdownItemVariants()`  here anymore?

Reply:
It's not an internal detail. It's the default styles for the dropdown item. You can import them and use them if you want to extend its styles, or omit it if you want to use it unstyled.

Reply:
Why do I have to do that if I just want to update some styles?

Reply:
If not mistaken customization comes from `style` function. Without it everything else is hardcoded, right?